doc/in-progress/sui-kiosk.md

```diff
@@ -0,0 +1,126 @@
+* LISTED EXCLUSIVELY - an item placed or locked in the kiosk by an extension that calls the  `kiosk::list_with_purchase_cap` function. Only the kiosk owner can approve calling the function. The owner can only borrow it immutably. The extension must provide the functionality to delist / unlock the asset, or it might stay locked forever. Given that this action is explicitly performed by the Owner - it is the responsibility of the Owner to choose verified and audited extensions to use.
+
+When someone purchases an asset from a kiosk, the asset leaves the kiosk and ownership transfers to the buyer’s address.
+
+## List items from your kiosk
+
+As a kiosk owner, you can use the [`list`](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/kiosk.md#0x2_kiosk_list) function to list any item you placed in the kiosk. Buyers can purchase only items that have an associated transfer policy ([`TransferPolicy`](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/transfer_policy.md) object) that is shared and available to them. Use the `list` function to list assets already placed in your kiosk.
+
+To list an item in a Sui kiosk:
+
+```rust
+kiosk::list(Kiosk, KioskOwnerCap, ID, price)
+```
+
+## List assets that include a PurchaseCap
+
+When you use the `list` function, you specify a fixed price that someone must pay to purchase it. The listing is available to anyone. To list an item without a set price, you can use the [`kiosk::list_with_purchase_cap`] (https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/kiosk.md#0x2_kiosk_list_with_purchase_cap) function to include a `PurchaseCap`. A `PurchaseCap`is an object that grants exclusive rights to purchase a specific asset. The asset gets a state of LISTED EXCLUSIVELY. You can’t take the asset from the kiosk, delist it, or modify it until the `PurchaseCap` is returned (which unlocks the asset) or the holder uses it to purchase the asset. You can use `PurchaseCap` to implement any special trading conditions you want, such as for an auction, or to send to a user directly.
+
+To list an asset with a `PurchaseCap`:
+
+```rust
+kiosk::list_with_purchase_cap(Kiosk, KioskOwnerCap, id, min_price)
+```
+
+Only the user that holds the `PurchaseCap` can purchase the associated asset.
+
+## Remove an item from your Sui kiosk
+
+To remove an item from a kiosk, as the kiosk owner, use the `kiosk::take` function. You can take only items that are not locked or listed. 
+
+To take an item from your Sui kiosk:
+
+```rust
+kiosk::take(Kiosk, KioskOwnerCap, ID): Item
+```
+
+## Purchase an item from a kiosk
+
+To purchase a listed item from a kiosk, use the [`kiosk::purchase`](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/kiosk.md#function-purchase) function. When your purchase transaction completes, the asset  transfers to your address and you become the asset owner.
```

Comment:
I would elaborate here, that no transfer happens. An asset is available to the sender once the TransferRequest is resolved. However, if there's a policy which enforces locking (such as kiosk lock rule), the asset needs to be placed into one's Kiosk.